Wabbajack.Installer/StandardInstaller.cs

```diff
@@ -231,6 +231,7 @@ private async Task WriteMetaFiles(CancellationToken token)
         await _configuration.Downloads.EnumerateFiles()
```

Comment:
Now that I think about it, this should probably be a `.Where(download => download.Extension == Ext.Meta)` for increased performance.